sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -53,6 +53,7 @@ import {
   SequenceControlService,
   WorkflowClientService,
 } from "@sims/services";
+import { IER12IntegrationModule } from "./institution-integration/ier-integration/ier12-integration.module";
```

Comment:
why is it part of app.module when the controller is defined in system access module?